packages/blade/src/components/Link/Link/Link.stories.tsx

```diff
@@ -0,0 +1,206 @@
+    },
+    {
+      themeTokenName: 'bankingTheme',
+      lightModeURL:
+        'https://www.figma.com/file/sAdplk2uYnI2ILnDKUxycW/Blade---Banking-Dark?node-id=10564%3A195587',
+      darkModeURL:
+        'https://www.figma.com/file/sAdplk2uYnI2ILnDKUxycW/Blade---Banking-Dark?node-id=10564%3A195587',
+    },
+  ]);
+
+  return (
+    <>
+      <Title />
+      <Subtitle>
+        This is the Link component which can be used for showing external or internal Links to the
+        user. The Link component can also be used as an inline button in certain cases with the
+        `button` variant
+      </Subtitle>
+      <StorybookLink withArrow={true} href={figmaURL} target="_blank" rel="noreferrer noopener">
+        View in Figma
+      </StorybookLink>
+      <br />
+      <br />
+      <Title>Usage</Title>
+      <Highlight language="tsx">{`import { Link } from '@razorpay/blade/components' \nimport type { LinkProps } from '@razorpay/blade/components'`}</Highlight>
+      <Description markdown="> **Note:** While using the `Link` component with React Native, please ensure you have gone through platform-specific prerequisites like adding `LSApplicationQueriesSchemes` in `Info.plist` for iOS and adding `intent` queries in `AndroidManifest.xml` for Android. For a detailed guide, follow React Native's [Linking Documentation](https://reactnative.dev/docs/linking#canopenurl)." />
+      <Title>Example</Title>
+      <Subtitle>
+        This is the default Link. You can change the properties of this Link using the controls in
+        the table below.
+      </Subtitle>
+      <Primary />
+      <Title>Properties</Title>
+      <ArgsTable story={PRIMARY_STORY} />
+      <Stories />
+    </>
+  );
+};
+
+export default {
+  title: 'Components/Link',
+  component: LinkComponent,
+  args: {
+    variant: 'anchor',
+    children: 'Learn More',
+    onClick: (event): void => {
+      console.log('clicked', event);
+    },
+    href: 'https://github.com/razorpay/blade',
+    target: '_blank',
+    rel: 'noreferrer noopener',
+  },
+  argTypes: {
+    icon: {
+      name: 'icon',
+      type: 'select',
+      options: Object.keys(iconMap),
+    },
+  },
+  parameters: {
+    docs: {
+      page: Page,
+    },
+  },
+} as Meta<LinkProps>;
+
+const LinkTemplate: ComponentStory<typeof LinkComponent> = ({ icon, children, ...args }) => {
+  const IconComponent = iconMap[(icon as unknown) as string];
+
+  return (
+    <LinkComponent icon={IconComponent} {...args}>
+      {children}
+    </LinkComponent>
+  );
+};
+
+export const Default = LinkTemplate.bind({});
+// Need to do this because of storybook's weird naming convention, More details here: https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#single-story-hoisting
+Default.storyName = 'Default';
+
+export const LinkButton = LinkTemplate.bind({});
+LinkButton.storyName = 'Link Button';
+LinkButton.args = {
+  variant: 'button',
```

Comment:
Cleaned it up. Ab nahi hoga hai issue in stories 🙈 

Lower API surface in this case would lead to our components behaving "smartly" which could confuse the consumers especially since the underlying element itself changes